tests/testthat/test-sas-results-part-1.R

```diff
@@ -0,0 +1,367 @@
+test_that("Posterior summaries for probabilities of
+          DLT (2-parameter logistic model) and recommended doses (NCRMLoss):
+          crmPack vs. SAS - Example 1", {
+  set.seed(0504201914)
+  mcmc_options <- McmcOptions(
+    burnin = 5000,
+    step = 2,
+    samples = 1000000,
+    rng_kind = "Wichmann-Hill",
+    rng_seed = 1
+  )
+
+  dose_grid_sc1 <- c(10, 20, 35, 50, 65, 80, 90, 100)
+
+  data <- Data(
+    x = c(rep(10, 3)),
+    y = c(rep(0, 3)),
+    cohort = c(rep(1, 3)),
+    doseGrid = dose_grid_sc1,
+    ID = 1:3
+  )
+
+  model_bcrm_sc1 <- LogisticLogNormal(
+    mean = c(-0.708, -0.389),
+    cov = matrix(c(1.2^2, 0, 0, 0.9^2), nrow = 2),
+    ref_dose = 90
+  )
+
+  ncrm_loss_sc1 <- NextBestNCRMLoss(
+    target = c(0.2, 0.35),
+    overdose = c(0.35, 0.6),
+    unacceptable = c(0.6, 1),
+    max_overdose_prob = 0.9999,
+    losses = c(1, 0, 2, 3)
+  )
+
+  postSamples <- mcmc(data, model_bcrm_sc1, mcmc_options)
+
+  dose_rec_loss <- expect_silent(nextBest(ncrm_loss_sc1,
+    doselimit = Inf,
+    postSamples, model_bcrm_sc1, data
+  ))
+
+  rec_dose_sc1 <- dose_rec_loss$value
+
+  prob_samples_mat <- matrix(
+    nrow = size(postSamples@options),
+    ncol = data@nGrid
+  )
+
+  # evaluate the probs, for all samples
+  for (i in seq_len(data@nGrid)) {
+    prob_samples_mat[, i] <- prob(
+      dose = data@doseGrid[i],
+      model_bcrm_sc1,
+      postSamples
+    )
+  }
+  pq75 <- apply(prob_samples_mat, 2, function(x) quantile(x, 0.75))
+
+  res_sc1 <- cbind(
+    LOSS = dose_rec_loss$probs[, "posterior_loss"],
+    PTARGET = dose_rec_loss$probs[, "target"],
+    POVEREX = dose_rec_loss$probs[, "excessive"],
+    POVERUN = dose_rec_loss$probs[, "unacceptable"],
+    POVER = rowSums(dose_rec_loss$probs[, c("excessive", "unacceptable")]),
+    PMEAN = dose_rec_loss$probs[, "mean"],
+    PQ75 = pq75
+  )
+
+  # Posterior summaries computed by SAS
+  temp <- read.csv2(paste0(getwd(), "/testdata/sc1_sit1.csv"),
```

Comment:
I think for this much better to use 
```suggestion
  temp <- read.csv2(test_path("testdata/sc1_sit1.csv"),
```
and similar for the other files you read

Reply:
and don't forget to add yourself to package authors as mentioned above :-)

Reply:
Thank you for the feedback, Daniel! Also just added myself to the authors :)